.github/workflows/ci-pytorch-dockers.yml

```diff
@@ -1,259 +0,0 @@
-name: Docker
```

Comment:
Why is this completely removed, from debugging?

Reply:
Thank you for pointing it out! Will revert the change